src/events/ResultPreviewsManagerEvents.ts

```diff
@@ -1,5 +1,15 @@
 import { ISearchResultPreview } from '../magicbox/ResultPreviewsManager';
 
+/**
+ * Executed when a {@link Suggestion} is focused before {@link PopulateSearchResultPreviews} is called to fetch more options.
+ */
+export interface IUpdateResultPreviewsManagerOptionsEventArgs {
+  /**
+   * How many milliseconds should a {@link Suggestion} be focused for before {@link PopulateSearchResultPreviews} is called.
+   */
+  delayBeforePopulate?: number;
```

Comment:
I find "populate" in the name a little abstract. What do you think of the names `displayAfterDuration`, `selectAfterDuration`?

Reply:
good idea, I'll change it to `displayAfterDuration`.